packages/document/docs/en/plugins/list/plugin-css-minimizer.mdx

```diff
@@ -27,10 +27,19 @@ import { PackageManagerTabs } from '@theme';
 You can register the plugin in the `rsbuild.config.ts` file:
 
 ```ts title="rsbuild.config.ts"
-import { pluginCssMinimizer } from '@rsbuild/plugin-css-minimizer';
+import {
+  pluginCssMinimizer,
+  CssMinimizerWebpackPlugin,
+} from '@rsbuild/plugin-css-minimizer';
 
 export default {
-  plugins: [pluginCssMinimizer()],
+  plugins: [
+    pluginCssMinimizer({
+      pluginOptions: {
+        minify: CssMinimizerWebpackPlugin.cssnanoMinify,
```

Comment:
This is an example of custom CSS minimizer, it should not replace the basic usage example.

We can add it add a new example.

Reply:
And `CssMinimizerWebpackPlugin.cssnanoMinify` is the default, which is not good for example.

Reply:
I think I am confused... Is the basic usage of `css-minimizer-webpack-plugin` not to switch the minimizer but to configure the minimizer options?

Reply:
The default value in rsbuild is 
```typescript
const defaultOptions = {
  minimizerOptions: {
    preset: [
      'default',
      {
        mergeLonghand: false,
      },
    ],
  },
};
```

Oh..I understand. default value cssnano both exists in rsbuild and webpack-css-minimizer-plugin... which makes me confused

Reply:
You do not need to add `minify: CssMinimizerWebpackPlugin.cssnanoMinify` to switch the minimizer, it is the default value.

Reply:
Yeah

Reply:
Can we change the defaultOptions to make it more clear?
```ts
const defaultOptions = {
  minify: CssMinimizerWebpackPlugin.cssnanoMinify,
  minimizerOptions: {
    preset: [
      'default',
      {
        mergeLonghand: false,
      },
    ],
  },
};
```